collector/collector/purge.py

```diff
@@ -21,12 +21,15 @@
     import uwsgi
     from uwsgidecorators import cron
 
+    PURGE_OLD_RECORDS = app.config.get("PURGE_OLD_RECORDS", False)
```

Comment:
I think this should default to `True`, which is the current default. Do you have a justification for changing the default?

Reply:
No, this is my mistake, what do you think the default value should be?